defaults/main.yml

```diff
@@ -96,6 +96,7 @@ k8s_ingress_certificate_issuer: letsencrypt-production
 # TLS certificate as Subject Alternative Name(s)
 k8s_ingress_tls_domains_extra: []
 k8s_ingress_tls_domains: "{{ k8s_domain_names + k8s_ingress_tls_domains_extra }}"
+k8s_ingress_class: nginx
```

Comment:
If the separate variable name is kept in https://github.com/caktus/ansible-role-k8s-web-cluster/pull/25#pullrequestreview-826433846, it might be nice to include a comment here about setting that variable, too.